File: scratch/test_ai.js
const fetch = require('node-fetch');

async function test() {
  const url = 'http://localhost:3000/api/ai/message';
  const payload = {
    occasion: 'wedding',
    relationship: 'friend',
    senderName: 'Agent',
    recipientName: 'User',
    tone: 'warm'
  };

  try {
    const response = await fetch(url, {
      method: 'POST',
      headers: { 'Content-Type': 'application/json' },
      body: JSON.stringify(payload)
    });
    const data = await response.json();
    console.log('Status:', response.status);
    console.log('Data:', JSON.stringify(data, null, 2));
  } catch (error) {
    console.error('Error:', error);
  }
}

test();

File: scratch/test_ai.py
import requests
import json

url = "http://localhost:3000/api/ai/message"
payload = {
    "occasion": "wedding",
    "relationship": "friend",
    "senderName": "Agent",
    "recipientName": "User",
    "tone": "warm"
}

try:
    response = requests.post(url, json=payload)
    print(f"Status: {response.status_code}")
    print(f"Response: {response.text}")
except Exception as e:
    print(f"Error: {e}")
